### ISSUE TITLE
Build a recruiting events & conference strategy 

### ISSUE THREAD

**Issue by jonathanbarker** [status-im/people-ops#7]

**Comment by rajanie** [comment]
Are you looking to also do University recruiting/outreach? A friend of mine does it for Ripple and they use [handshake](https://app.joinhandshake.com/login) to get interns / students aware of programs they have. Might be something to look into.

**Comment by jonathanbarker** [comment]
Yes for sure. I think it's less of a hiring priority right now, but I've put a few notes together https://notes.status.im/Zbb8M7I4QQW7R8iOYkNMwg?both

**Comment by rajanie** [comment]
I can add some top schools in the US with Blockchain / decentralized programs

**Comment by jonathanbarker** [comment]
OKR on hold since Status is not recruiting right now.
